_posts/2017-11-29-utdta-talk.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: tomesh @ UofT Decentralized Tech Association
```

Comment:
We tend not to stylize "tomesh" in paragraphs, 
Maybe the title could be "Talk at U of T.... Meetup"